singleobject/reader.go

```diff
@@ -0,0 +1,59 @@
+package singleobject
```

Comment:
I think the convention for naming would be `single_object`, but I'd also be ok with `soe` for this package.

Reply:
renaming done

Reply:
done